src/translations/locales/en/common.json

```diff
@@ -293,7 +340,12 @@
   "error": {
     "transactionFailed": {
       "timeOut": "Transaction timed out",
-      "default": "Transaction failed"
+      "default": "Transaction failed",
+      "offline": "You can't send transaction while offline",
+      "notEnoughGas": "Not enough gas to cover the transaction fee. Top up your ETH balance",
+      "notOwnedCollectible": "You do not longer own this collectible",
+      "collectibleCantBeTransferred": "This collectible can not be transferred",
```

Comment:
If this is a good moment for corrections, these two messages don't look right to me. "do no longer own" and "can't/cannot be transferred"?

Reply:
Oh, true. Didn't notice those, to be honest ;/
Will adjust those. Thanks!